sled-agent/src/bin/sled-agent-overlay-files.rs

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use omicron_sled_agent::bootstrap::trust_quorum::{
+    RackSecret, ShareDistribution,
+};
+
+use anyhow::{anyhow, Result};
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "sled-agent-overlay-files",
+    about = "Generate server unique files for deployment"
```

Comment:
The reason I used a generic name is because we may generate other distinct files in the sled agent, and would have a similar binary for different crates. I'm not really against this change, just laying out my rationale.

Will consider it in a follow up.

Reply:
I added a doc comment for the file.